packages/frontend/core/src/desktop/dialogs/setting/workspace-setting/index.tsx

```diff
@@ -22,16 +30,20 @@ export const WorkspaceSetting = ({
 }) => {
   switch (activeTab) {
     case 'workspace:preference':
+      return <WorkspaceSettingDetail onCloseSetting={onCloseSetting} />;
```

Comment:
this is strange, what if user switched to 'workspace:properties' and close the panel, the `onCloseSetting` will never be called?

Reply:
The `onCloseSetting` prop name may be misleading. It is actually a handler for closing the setting modal, not the panel itself.